contrib/windows/msys_build.sh

```diff
@@ -206,3 +206,4 @@ cat Make.user
 make -j3 VERBOSE=1
 make build-stats
 #make debug
+make prefix=/c/projects/julia/usr VERBOSE=1 -C examples
```

Comment:
I figured I should run the installed version, but it would work with either I think. This line is the only way it's been able to run any julia executable at all so far, because I have no idea where anything is in this configuration.